src/app/review-item/components/question/question.component.html

```diff
@@ -1,5 +1,8 @@
-<div class="question" [class.no-border]="isChild">
-  <div class="question__title">{{ title }}</div>
+<div class="question" [class.no-border]="isChild" [attr.id]="question.question_id">
+  <div class="question__title">
+    {{ title }}
+  </div>
+  <div class="question__error" *ngIf="questionForm.invalid && questionForm.touched">Du hast diese Frage noch nicht beantwortet.</div>
```

Comment:
I find it hard to immediatly find the error among the other text.
Maybe the design team has an idea.

![image](https://user-images.githubusercontent.com/38329766/142931236-4c9f77fe-27c4-485b-8168-454f0f48c1c4.png)

Reply:
How about using red text for the headline (in this case "Frage 2") to make it more obvious?

Reply:
Like this?

![image](https://user-images.githubusercontent.com/88544124/143054717-c2fcc91d-253d-4b6b-bf6c-c811138b1b20.png)

Making the radio buttons red also seems not a good solution to me IMHO.

Reply:
What do you think of a red border around the entire question in combination with the text? 🤔 

![image](https://user-images.githubusercontent.com/38329766/143068563-28d447f6-8b1f-4c30-9634-fdd1c5d13a04.png)

Reply:
@rewodev Yes, thats what I meant. It's hard to estimate, whether this has a real effect, when I can only see this one question. What do you think? 

@SandroGlueck That looks good - I would then put the warning text on top - directly below the top border.

Reply:
Ok?

![image](https://user-images.githubusercontent.com/88544124/143085854-037a288f-13a8-4469-a3df-dcb435985578.png)

Reply:
Looking awesome - Thanks! :)